addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -2420,16 +2420,29 @@ export function rgbToHex(rgb = '') {
             .join('')
     );
 }
-
-export function getRangePosition(el, document, options = {}) {
+/**
+ * Returns position of a range in form of object (end
+ * position of a range in case of non-collapsed range).
+ *
+ * @param {HTMLElement} el element for which range postion will be calculated
+ * @param {Document} document
+ * @param {Object} [options]
+ * @param {Number} [options.marginRight] right margin to be considered
+ * @param {Number} [options.marginBottom] bottom margin to be considered
+ * @param {Number} [options.marginTop] top margin to be considered
+ * @param {Number} [options.marginLeft] left margin to be considered
+ * @param {DOMRect} [options.parentContextRect] context rect from parent
+ * @returns {Object | undefined}
+ */
+export function getRangePosition(el, document, { marginRight, marginBottom, marginTop, marginLeft, parentContextRect } = {}) {
```

Comment:
We can't change the API in stable like this, in case someone was using it in their customization. Please only change the docstring without changing the arguments of the function.

Reply:
Thanks for letting me know. I'll keep that in mind :+1: .